cmds/core/chmod/chmod.go

```diff
@@ -75,7 +78,7 @@ func calculateMode(modeString string) (mode os.FileMode, octval uint64, mask uin
 	// `a=` is a valid (but destructive) operation. Do not turn a typo into that.
 	reMode = regexp.MustCompile("^[rwx]*$")
 	if len(m) < 3 || !reMode.MatchString(m[3]) {
-		return mode, octval, mask, fmt.Errorf("unable to decode mode %q. Please use an octal value or a valid mode string", modeString)
+		return mode, octval, mask, fmt.Errorf("%w:unable to decode mode %q. Please use an octal value or a valid mode string", strconv.ErrSyntax, modeString)
```

Comment:
```suggestion
		return mode, octval, mask, fmt.Errorf("unable to decode mode %q. Please use an octal value or a valid mode string: %w", modeString, strconv.ErrSyntax)
```

Reply:
same here, if you look at the output, it just reads better, which is why I did it this way. I realize there is a convention, but it is a guideline, not a rule? OK to leave this as is?